src/output/help.rs

```diff
@@ -595,12 +598,13 @@ impl<'help, 'app, 'parser, 'writer> Help<'help, 'app, 'parser, 'writer> {
             let pvs = a
                 .possible_vals
                 .iter()
-                .map(|&pv| {
-                    if pv.contains(char::is_whitespace) {
-                        format!("{:?}", pv)
+                .filter_map(|value| match value {
```

Comment:
Can't we do `if value.is_hidden()` instead of this match statement with destructuring?